src/cc2olx/models.py

```diff
@@ -272,6 +273,25 @@ def get_resource_content(self, identifier):
                     )
                     raise
                 return "html", {"html": html}
+            elif 'web_resources' in str(res_filename):
+                if imghdr.what(str(res_filename)):
```

Comment:
By combining these two `if` statements in single one like `elif 'web_resources' in str(res_filename) and imghdr.what(str(res_filename))` you can avoid code duplicating below:

```python
print("*** Skipping webcontent: {}".format(str(res_filename)))
return None, None
```

Reply:
Yes, I agree that these two conditions can be combined.
Actually, I put those two conditions separately because when I will be contributing for other webcontent conversion which are also from web_resources then there will be need for separate conditions.
But right now there is no need for separate conditions so I combined them for now and removed the else part.